docs/rules/arrow-parens.md

```diff
@@ -58,7 +58,7 @@ String options are:
 
 Object properties for variants of the `"as-needed"` option:
 
-* `"requireForBlockBody": true` modifies the as-needed rule in order to require parens if the function body is in an intructions block (surrounded by braces).
+* `"requireForBlockBody": true` modifies the as-needed rule in order to require parens if the function body is in an instructions block (surrounded by braces).
```

Comment:
I'm a bit confused on what this is actually trying to say, to be honest. Is this saying if the arrow body is a block with statements in it? If so, I think I would prefer "statement block" or "block with statements"...